webapp/stores/user_store.jsx

```diff
@@ -105,6 +105,10 @@ class UserStoreClass extends EventEmitter {
         this.currentUserId = user.id;
         global.window.mm_current_user_id = this.currentUserId;
         if (LocalizationStore.getLocale() !== user.locale) {
+            if (!GlobalActions) {
```

Comment:
Seems like that chain should be broken between TextFormatting and UserStore instead. Why does text formatting need the UserStore? Shouldn't it just be passed any info it needs? So it's independent?